internal/kafka/command_topic_consume.go

```diff
@@ -28,15 +27,19 @@ func (c *command) newConsumeCommand() *cobra.Command {
 		Args:              cobra.ExactArgs(1),
 		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
 		RunE:              c.consume,
-		Annotations:       map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLogin},
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: `Consume items from topic "my-topic" and press "Ctrl-C" to exit.`,
 				Code: "confluent kafka topic consume my-topic --from-beginning",
 			},
+			examples.Example{
+				Text: `Consume from a cloud Kafka topic named "my_topic" without logging in to Confluent Cloud.`,
+				Code: "confluent kafka topic consume my_topic --api-key my-key --api-secret my-secret --bootstrap SASL_SSL://pkc-abc12:9092 --value-format avro --schema-registry-endpoint https://psrc-ab123 --schema-registry-api-key my-sr-key --schema-registry-api-secret my-sr-secret",
+			},
 		),
 	}
 
+	cmd.Flags().String("bootstrap", "", `Comma-separated list of broker hosts, each formatted as "host" or "host:port". For Confluent Cloud, it's Kafka cluster endpoint.`)
```

Comment:
I think you want a `StringSlice` here, so users can optionally pass multiple instances of `--bootstrap`

Reply:
> Comma-separated list of broker hosts

This is not the case for Confluent Cloud. Can we rewrite this to something like "Kafka cluster endpoint (Confluent Cloud) or list of broker hosts (Confluent Platform)"?

Reply:
as it's not the case for cloud, i think it's better to leave it a string

Reply:
`Kafka cluster endpoint (Confluent Cloud); or comma-separated list of broker hosts (Confluent Platform), each formatted as "host" or "host:port".`

Reply:
how does it look